src/Logger.php

```diff
@@ -19,22 +19,16 @@ class Logger implements LoggerInterface
 {
     use LoggerTrait;
 
-    /**
-     * @var Raven_Client
-     */
+    /** @var Raven_Client */
     protected $client;
-    /**
-     * @var SenderInterface
-     */
+
+    /** @var Sender|SenderInterface */
```

Comment:
We accept the interface in the constructor, so the implementation `Sender` should not be considered.

Reply:
Does phpstan throws errors?